getimage.py

```diff
@@ -0,0 +1,48 @@
+import httplib, urllib, base64, random, string, json, pprint, cStringIO
+from PIL import Image
+import io 
+
+def get_random_image(): 
+    search_term = "".join(random.choice(string.ascii_lowercase) for j in range(random.randint(1,6)))
```

Comment:
Explicit variables should be used instead of single characters **(j)**.
Try using 'xrange' instead of 'range' when iterating over the arbitrarily sized array.